typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/presentation.ts

```diff
@@ -274,29 +274,62 @@ function buildOutput(output: string[]): string {
 /**
  * Returns the graph for all the nodes.
  *
- * Will insert the `rootItem` by default as a start node an search for other start nodes
+ * Inserts nodes for the default Start and End workflow elements (not actual WF items).
+ * Will insert the `rootItem` (or first node, if not explicitly annotated)
+ * and any secondary start nodes (e.g. Default Error Handler) as targets of the default Start node.
+ * Secondary start nodes will be placed at the same level (column) as the default Start node.
  *
- * Will insert the default `end` node as well as the others require it but it's not an actual workflow item
+ * Remaining items are placed acording to their targets.
+ * @param {WorkflowDescriptor} workflow
+ * @returns {Graph}
+ * @throws Error when there are targets missing or misspelled or when a secondary start node is targeted.
  */
-function getGraph(workflow: WorkflowDescriptor) {
-	// start node
-	const startNodes = [`item${findItemByName(workflow.items, workflow.rootItem) || "1"}`];
-	const nodes = workflow.items.map((item, i) => {
-		switch (item.strategy.getCanvasType()) {
-			case "error-handler":
-				startNodes.push(`item${i + 1}`);
-				break;
-			default:
-				break;
-		}
+function getGraph(workflow: WorkflowDescriptor): Graph {
+	let errors = [];
+	const nodes: GraphNode[] = workflow.items.map((item, i) => getGraphNode(item, i, errors));
+	if (errors.length) {
```

Comment:
You should fail immediately. At this point on the program is in degraded state with no chance of recovery, exit fast

Reply:
Will throw on first error.